rasa/core/policies/unexpected_intent_policy.py

```diff
@@ -552,7 +552,8 @@ def _should_skip_prediction(tracker: DialogueStateTracker) -> bool:
                  from getting stuck in a prediction loop.
                 For example, if the last `ActionExecuted` event
                 contained `action_unlikely_intent` predicted by
-                `IntentTEDPolicy` and if `IntentTEDPolicy` runs inference
+                `UnexpecTEDIntentPolicy` and
```

Comment:
pls also fix `_metadata_filename`

Reply:
do you want to rename `class IntentTED` to `ExpecTED` as well?

Reply:
> do you want to rename class IntentTED to ExpecTED as well?

No, `ExpecTED` does not make sense at all. `IntentTED` is good to indicate that the architecture is TED but for intents.